docs/topics/proc_deploying-the-secured-booster-using-the-oc-cli-client.adoc

```diff
@@ -72,6 +73,6 @@ $ npm install && npm run openshift -- \
       -d SSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='{"https://"}{.spec.host}{"/auth\n"}')
 ----
 
-These commands install any missing module dependencies, then using the xref:about-nodeshift[Nodeshift] module, deploy the booster on OpenShift.
+These commands install any missing module dependencies, then using the xref:about-nodeshift[Nodeshift] module, deploy the example on OpenShift.
```

Comment:
deploy the example on OpenShift.--> deploy the example application on OpenShift.